docs/online/src/scenarios/postmessage.rst

```diff
@@ -269,6 +270,76 @@ You can send the following messages; all others are ignored:
         }
 
 
+..  data:: Host_RefreshSessionInfo
+
+    When the host wants to send new tokens to |wac|, for instance when an :term:`access token` is about to expire, 
+    a :data:`Host_RefreshSessionInfo` message is posted. The host can send a set of new access tokens and their properties 
+    in this post message. 
+
+    ..  note:: 
+        |wac| expects a :data:`Host_RefreshSessionInfo` response to a :data:`App_RefreshSessionInfo` request. If no response is 
+        received by |wac|, the session could expire. 
+    
+    If the host is unable to issue new tokens an error message must be provided, with an optional description of the 
+    reason. If no errors are present, the value property must contain the token and its expiry. 
+
+    ..  note::
+        The host must indicate the support for :data:`App_RefreshSessionInfo` by setting the :term:`SupportsSessionRefresh` parameter 
+        in the :ref:`CheckFileInfo` response to true.
+
+    .. attribute:: Values
+        :noindex:
+
+        Tokens  *(JSON-formatted list)*
+            :term:`Tokens` refers to an array of *(JSON-formatted object)* identifying new :term:`access token` sent to |wac|.
+
+            Name *(string)*
+            :term:`Name` refers to the property used to identify token elements.
+
+            Expiry *(long)*
+            :term:`Expiry` refers to the time the token would no longer be valid.
```

Comment:
You need to explain more about this. What is the format? Milliseconds? Nanoseconds? What's the epoch?